helm/cluster-cloud-director/templates/kubeadmcontrolplane.yaml

```diff
@@ -109,6 +109,7 @@ spec:
       {{- if $.Values.network.staticRoutes }}
       {{- include "staticRoutes" . | nindent 6}}
       {{- end }}
+      {{- include "ntpServers" . | nindent 4}}
```

Comment:
@erkanerol This is correct obviously but I find it a bit confusing as the `ntp:` section name is defined in the template, here it looks like it is under `files:`. 

As a personal preference, I'd rather see the `ntp:` section explicitly here. But it's only cosmetic.

Reply:
But then we need to add `if-else` logic here, which makes this file more complex. I prefer encapsulating features with helper functions.

Reply:
moving the whole `{{- include "ntpServers" . | nindent 4}}` 2 spaces to the left should definitely make it more readable. Now it's under files section, but correctly indented by 4 spaces

Reply:
Totally agree and fixed.